src/TwitterShareButton.ts

```diff
@@ -5,10 +5,16 @@ import createShareButton from './hocs/createShareButton';
 
 function twitterLink(
   url: string,
-  { title, via, hashtags = [] }: { title?: string; via?: string; hashtags?: string[] },
+  {
+    title,
+    via,
+    hashtags = [],
+    related = [],
+  }: { title?: string; via?: string; hashtags?: string[]; related?: string[] },
 ) {
   assert(url, 'twitter.url');
   assert(Array.isArray(hashtags), 'twitter.hashtags is not an array');
+  assert(Array.isArray(hashtags), 'twitter.related is not an array');
```

Comment:
This is supposed to be `Array.isArray(related)`, right?

Reply:
You are correct. Fixed!